helpers/base-helper.php

```diff
@@ -86,6 +86,16 @@ public function get_priority(): int {
 	public function has_async_content(): bool {
 		return $this->has_async_content ?? false;
 	}
+	/**
+	 * Indicates whether the helper should initally load inline
+	 *
+	 * @since 0.0.1
```

Comment:
We are marking the new methods as `0.0.2`. The `0.0.1` methods come from DotCom.